src/wizard/unreal/index.md

```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
 ___PUBLIC_DSN___
```

Comment:
I can't access these lines from here:

- Line 10, can you change `Plugins` to "Plugins"
- Line 12, put Settings > Plugins > Code Plugins in bold (not the word _menu_ though)
- Line 22, put change Project Settings to just Settings and put that nav path in bold and not code font
-